### PR TITLE
update universal architecture

### PR DESCRIPTION
**Pull request by pdarde-daitan**
Update universal architecture and remove obsolete files

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -13,3 +13,4 @@ target/
 
 # NPM
```
Comment (ecarrenho-daitan) [inline .gitignore:14]: Make NPM ignores similar to what we have in JIRA repo.

File: local-run/application.yaml.template [hunk 2]
```diff
@@ -1,10 +0,0 @@
-spring:
```
Comment (ecarrenho-daitan) [inline local-run/application.yaml.template:1]: keep this file

File: local-run/env.sh.sample [hunk 3]
```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
```
Comment (ecarrenho-daitan) [inline local-run/env.sh.sample]: keep this file.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  },
+  "dependencies": {
+    "babel-polyfill": "^6.23.0",
+    "react": "^15.3.2",
+    "react-dom": "^15.4.2",
+    "react-redux": "^5.0.2",
+    "react-router": "^3.0.2",
+    "redux": "^3.6.0",
+    "redux-saga": "^0.14.3",
+    "symphony-integration-commons": "file:///home/pdarde/symphony/app/App-Integrations-FE-Commons"
```
Comment (ecarrenho-daitan) [inline package.json]: Add actual version number.

File: run.sh [hunk 5]
```diff
@@ -1,22 +0,0 @@
-#!/bin/bash
```
Comment (ecarrenho-daitan) [inline run.sh]: keep this file.

File: src/main/webapp/html/app.html [hunk 6]
```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hello World App - View</title>
```
Comment (ecarrenho-daitan) [inline src/main/webapp/html/app.html]: We should add a proper title here... and on other configurator apps as well.

File: src/main/webapp/html/controller.html [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+<!-- This is the controller, which runs as a persistent iFrame, out of the user's view -->
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Hello World App - Controller</title>
```
Comment (ecarrenho-daitan) [inline src/main/webapp/html/controller.html]: We should add a proper title here... and on other configurator apps as well.

File: .gitignore (hunk 1 above)
Comment (pdarde-daitan) [reply .gitignore:14]: Done.

File: local-run/application.yaml.template (hunk 2 above)
Comment (pdarde-daitan) [reply local-run/application.yaml.template:1]: Done.

File: local-run/env.sh.sample (hunk 3 above)
Comment (pdarde-daitan) [reply local-run/env.sh.sample]: Done.

File: package.json (hunk 4 above)
Comment (pdarde-daitan) [reply package.json]: Today we have the version "0.11.0-SNAPSHOT", as well as Jira. Which should be the correct version?

File: run.sh (hunk 5 above)
Comment (pdarde-daitan) [reply run.sh]: Done.

File: src/main/webapp/html/app.html (hunk 6 above)
Comment (pdarde-daitan) [reply src/main/webapp/html/app.html]: Done.

File: src/main/webapp/html/controller.html (hunk 7 above)
Comment (pdarde-daitan) [reply src/main/webapp/html/controller.html]: Done.

**Pull request merged**